include/conversation.php

```diff
@@ -1488,6 +1490,24 @@ function conv_sort(array $item_list, $order)
 	return $parents;
 }
 
+/**
+ * @brief usort() callback to sort item arrays by pinned and the received key
+ *
+ * @param array $a
+ * @param array $b
+ * @return int
+ */
+function sort_thr_pinned_received(array $a, array $b)
```

Comment:
Plz rename `$a` and `$b` to more meaningful names and explain what -1 / 1 as return mean

Reply:
I just took the existing field names. This is a helper function for the sort function. It needs these integer values to to the correct sorting.

Reply:
"Meaningful" would possibly be "field1" and "field2" - which doesn't tell more.

Reply:
`$a` and `$b` are standard callback function parameters for `usort`: https://www.php.net/manual/en/function.usort.php